app/views/course/assessment/answer/_worksheet.html.slim

```diff
@@ -1,16 +1,22 @@
 - if answer.submission.attempting? && answer.specific.is_a?(Course::Assessment::Answer::Programming)
+  - programming_answer = answer.specific
+  - if answer.question.actable.attempt_limit
+    - button_text = t('.run_code_with_limit', limit: programming_answer.attempting_times_left)
+  - else
+    - button_text = t('.run_code')
```

Comment:
@allenwq Yup I think we can do that - maybe just add the message into the placeholder? Sorry was trying to figure out if there are other methods but I cannot think of any >_<